src/style/typography.scss

```diff
@@ -5,8 +5,8 @@
 
 /// Constants ------------------------------------------------------------------
 
-$iui-sans: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$iui-monospace: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+$iui-sans: 'Open Sans', BlinkMacSystemFont, -apple-system, 'Segoe UI', Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', sans-serif;
+$iui-monospace: SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace;
```

Comment:
- [SFMono](https://github.com/supercomputra/SF-Mono-Font) says "Please note that it is forbidden to use this typeface for your commercial project."
- Menlo seems to have been phased out completely. Monaco is even older.
And I don't think any of the above 3 fonts are even allowed to be used in non-Apple contexts. Likewise, Consolas is Windows-specific, though it has fewer restrictions.

Shouldn't we use a better _default_ font that looks the same on all operating systems?

Reply:
I don't understand how [Putra Z.](https://github.com/supercomputra) owns SFMono because [San Francisco Mono is an Apple built font](https://developer.apple.com/fonts/). 🤔

For the monospaced font stack, maybe the first font should be a web font similar to Open Sans so it will be consistent across all OSes.  I like [Inconsolata](https://fonts.google.com/specimen/Inconsolata?category=Monospace).